src/components/analyses/details/ParamsPanel.js

```diff
@@ -27,9 +47,72 @@ import {
     Typography,
 } from "@material-ui/core";
 
+function InputParameterValue(props) {
+    const { id, param_type, path } = props;
+
+    const [statError, setStatError] = React.useState();
+
+    React.useEffect(() => {
+        if (!path) {
+            setStatError(null);
+        }
+    }, [path]);
+
+    const displayValue = parseNameFromPath(path);
+    const linkTarget = getFolderPage(
+        AppParamTypes.FOLDER_INPUT === param_type ? path : getParentPath(path)
+    );
+
+    const { t } = useTranslation("analyses");
+
+    const { isFetching: isFetchingStat } = useQuery({
+        queryKey: [DATA_DETAILS_QUERY_KEY, { paths: [path] }],
+        queryFn: getResourceDetails,
+        config: {
+            enabled: path,
+            onSuccess: () => {
+                setStatError(null);
+            },
+            onError: (error) => {
+                if (ERROR_CODES.ERR_DOES_NOT_EXIST === getErrorCode(error)) {
+                    setStatError(
+                        t("errorInputDoesNotExist", { path: linkTarget })
+                    );
+                } else {
+                    setStatError(null);
+                }
+            },
```

Comment:
Should we show error that happens when irods is or data-info is down ? 🤔

Reply:
Not sure 🤔 
This was basically copy-paste from the app launch input selectors, which displayed "not found" errors in the launch form the same way (and this is the way they were displayed for input params in the old DE too). I guess it silently ignores anything except "not found" errors, and if there is still a problem when the app launches, the analysis will fail then:
https://github.com/cyverse-de/sonora/blob/0b03f7314c717e646339f1ac681bc8058aefdec5/src/components/apps/launch/params/DiskResourceSelector.js#L30-L54

In this display, I could add a different error message if the path couldn't be checked.
Then maybe the file could still display as a link so the user can check if the error is temporary by clicking on it?

Reply:
It turned out to be pretty easy to also display other errors in an `ErrorTypographyWithDialog`, so I'll show that, but also keep the link in case the error is temporary (sci-analyst feedback also said that should be fine).

![Analysis params input - other errors](https://user-images.githubusercontent.com/996408/118569411-c23ba700-b72e-11eb-8ca6-630e8799acb5.png)

This coud also be updated in the app launch form's default input params (also relaunch input params) in a future PR.

Reply:
Actually, this should never be an issue in the app launch form, since the service will remove inputs that no longer exist (or the endpoint will return an error for other stat errors). This could still be useful when checking default inputs in the app editor, though.